rabbit_mq/main.tf

```diff
@@ -78,3 +78,13 @@ resource "aws_mq_broker" "mq" {
   subnet_ids          = (var.publicly_accessible ? null : (var.deployment_mode != "SINGLE_INSTANCE" ? module.subnets.subnets.ids : [random_shuffle.random_subnet.result[0]]))
 }
 
+resource "aws_secretsmanager_secret" "broker_password" {
+  name        = "${var.broker_name}-amq-password"
```

Comment:
`name        = "${var.broker_name}-mq-password"`

amq is different. This module is actually for rmq. I'd stick with mq.